README.md

```diff
@@ -216,6 +216,22 @@ redactor.redactAsync('I live at 123 Park Ave Apt 123 New York City, NY 10002 and
 });
 ```
 
+#### Google DLP content size limit
+
+The Google DLP service has a content size limit of 524288. If the input is over this limit, the `GoogleDLPRedactor` will 
```

Comment:
characters?

Reply:
yes, characters

Reply:
Actually, turns out it's bytes and there is a bug with this implementation since `.length` counts unicode characters. I need to calculate the number of bytes used instead.